Detections/SecurityEvent/RDP_Nesting.yaml

```diff
@@ -97,27 +97,34 @@ query: |
   ) on Account, $left.SecondComputer == $right.Computer, $left.SecondIPAddress == $right.IpAddress
   | summarize FirstHopFirstSeen = min(FirstHop), FirstHopLastSeen = max(FirstHop) by Account, FirstComputer, FirstIPAddress, SecondHop, SecondComputer,
   SecondIPAddress, AccountType, Activity, LogonTypeName, ProcessName
-  | extend timestamp = FirstHopFirstSeen, AccountCustomEntity = Account, HostCustomEntity = FirstComputer, IPCustomEntity = FirstIPAddress
+  | extend timestamp = FirstHopFirstSeen
+  | extend AccountName = tostring(split(Account, "@")[0]), AccountUPNSuffix = tostring(split(Account, "@")[1]) 
+  | extend HostName = tostring(split(FirstComputer, ".")[0]), DomainIndex = toint(indexof(FirstComputer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(FirstComputer, DomainIndex + 1), FirstComputer)
 entityMappings:
   - entityType: Account
     fieldMappings:
-      - identifier: FullName
-        columnName: AccountCustomEntity
+      - identifier: Name
```

Comment:
Account is actually a Name + NTDomain in this case.  So you can map it, but you have to split it as follows:

| extend AccountName = tostring(split(Account, "\\")[1]), AccountNTDomain = tostring(split(Account, "\\")[0])